.env.template

```diff
@@ -1,10 +1,8 @@
 # these values are used in the local docker env. You can use "localhost" hostname if you run the application without docker 
 POSTGRES_HOSTNAME=postgres_bloom
-POSTGRES_HOSTNAME_OUTSIDE_WHEN_DOCKER=localhost # when using docker, the hostname is mapped to the outside world
```

Comment:
Hi
I don't really understand the use of these POSTGRES_HOSTNAME_OUTSIDE_WHEN_DOCKER and POSTGRES_PORT_OUTSIDE_WHEN_DOCKER that semms to be specific for Docker use but they are finally used in bloom.config file and this file is not specific to Docker use but is also used for host native installation